client/components/data/query-top-posts/README.md

```diff
@@ -0,0 +1,65 @@
+Query Top Posts
```

Comment:
It would be nice to include some sort of a live-doc that could be used by the reviewer to quickly make sure the `QueryTopPosts` component worked as expected. Maybe something that would show in https://wpcalypso.wordpress.com/devdocs